config/load.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2019-2020 Solus Project
+// Copyright © 2019-Present Solus Project
```

Comment:
A copyright header indicates the start of copyright, so you know when it enters public domain. `Present` is therefore not a valid value. The rule is that you update the last year every time you edit a file significantly.

Reply:
No one can seem to agree on this; other large projects (I think Fedora) says that the dates are completely irrelevant, and they have a lot of lawyers.

Reply:
This was my source: https://hynek.me/til/copyright-years/

Reply:
I think the thing no one can seem to agree on is if a statement like this is actually required. As far as I know, it's not.

This means that the copyright statement in it's entirety is informative: the dates in the copyright statement can be used to inform receivers of the source code in the future of the copyright status. This is more relevant when the source is not available in a Git repo with dates and authors.

I'm all for just removing the years entirely by the way, it's just that having *present* there defeats the purpose of the years in the copyright statement.

Reply:
React used to use "present" before removing the year range altogether. Both are probably sort of valid, but at this point let's embrace your suggestion and the React way.